docs/search/index.mjs

```diff
@@ -2,14 +2,17 @@ import algoliasearch from 'algoliasearch';
 import fs from 'fs/promises';
 
 const indexJson = JSON.parse(await fs.readFile('./index.json'));
+const discussionsJson = JSON.parse(await fs.readFile('./discussions.json'));
 
 const key = process.env['ALGOLIA_KEY'];
 if ( ! key ) { throw Error(`No key!`); }
 const client = algoliasearch('M53HHHS0ZW', key);
 const index = client.initIndex('docs');
+const discussionsIndex = client.initIndex('discussions');
```

Comment:
I do not think you should make a new index, but you should include the data in the same index

Reply:
@TheChronicMonster wanted it in a separate index.

Reply:
[504dd82](https://github.com/reach-sh/reach-lang/pull/1113/commits/504dd82ab4c7058b4a69ac651252dd97b3ef6151)